src/app.ts

```diff
@@ -91,4 +91,35 @@ app.get('/flights', (req, res) => {
   res.json(flights);
 });
 
+// /airport
+// Retrieve a list of details for a given airport
+app.get('/airport', (req, res) => {
+  const codeFormatText = 'AAA';
+  const { query } = req;
+
```

Comment:
Let's use TypeScript's optional chaining and check for `query`, `query.code`, and then uppercase the string. If either of the first two checks fail, `airportCode` will be undefined
```suggestion
  const airportCode = query?.code?.toUpperCase();

```